hoodie-client/src/test/java/com/uber/hoodie/TestHoodieClientOnCopyOnWriteStorage.java

```diff
@@ -316,6 +345,109 @@ public void testUpsertsWithFinalizeWrite() throws Exception {
     testUpsertsInternal(hoodieWriteConfig);
   }
 
+  @Test
+  public void testUpsertPreppedRecords() throws Exception {
```

Comment:
Can we just make some tests above do insertPreppedRecords/bulkInsertPreppedRecords instead of repeating code here? 

we can then pare this test down to just tagLocation and upsertPreppedLocation apis alone..

Reply:
in testUpsertPreppedRecords, we also need insertPreppedRecords and bulkInsertPreppedRecords  to prepare the test dataset, hence in testUpsertPreppedRecords we have also tested insertPreppedRecords & bulkInsertPreppedRecords and also ppsertPreppedRecords.  Maybe we should have a new name for the test method?

Reply:
My concerns were in the same vein. These APIs are pretty light, so changing few existing test cases to call insertPreppedRecords instead of insert() or bulkInsertPreppedRecords instead of bulkInsert() should be sufficient to test them?